types/lnrpc.d.ts

```diff
@@ -282,8 +282,8 @@ export interface ChannelBackups {
 }
 
 export interface ChanBackupSnapshot {
-  singleChanBackups?: ChannelBackups;
-  multiChanBackup?: MultiChanBackup;
+  singleChanBackups: ChannelBackups;
```

Comment:
Both fields don't seem to be required everywhere `ChanBackupSnapshot` is referenced. The `verifyChanBackup` usage looks like it's just one or the other. `Partial<ChanBackupSnapshot>` can be used in that case.